app/dashboard/views/mailbox.py

```diff
@@ -112,12 +147,12 @@ def mailbox_route():
                 elif not email_can_be_used_as_mailbox(mailbox_email):
                     flash(f"You cannot use {mailbox_email}.", "error")
                 else:
-                    new_mailbox = Mailbox.create(
+                    transfer_mailbox = Mailbox.create(
```

Comment:
We shouldn't rename `new_mailbox` to `transfer_mailbox` here right?

Reply:
I agree. Done here https://github.com/simple-login/app/pull/1547